modules/oadp-gcp-wif-cloud-authentication.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-gcp-wif-cloud-authentication_{context}"]
+= Google WIF cloud authentication
+
+Applications running outside Google Cloud use service account keys to access Google Cloud resources, these service account keys can become a security risk if they are not managed correctly.
+
+With identity federation (WIF), you can use Identity and Access Management (IAM) to grant external identities IAM roles, including the ability to impersonate service accounts. This eliminates the maintenance and security risks associated with service account keys.
+
+WIF handles encrypting and decrypting certificates, extracting user attributes, and validation. WIF externalizes authentication, passing it over to Security Token Services (STS) and reduces the demands on individual developers. Authorization and controlling access to resources remain the responsibility of the application.
+
+[NOTE]
+====
+Google WIF cloud authentication is available for OADP 1.3.x and later.
+====
+
+If you do not use Google WIF cloud authentication, continue to xref:../../..//backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc#oadp-installing-operator-doc[Installing the OADP Operator].
+
+.Prerequisites
+
+* You must install a cluster in manual mode with link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/authentication_and_authorization/index#gcp-workload-identity-mode-installing[GCP Workload Identity configured].
```

Comment:
Can you try to use xrefs instead of links... also why link to 4.13

Reply:
I'll see if it can be done. 4.14 has not yet been published, if I link to it the link doesn't work.

Reply:
@anarnold97 
We can only use xrefs in assemblies. This reference is in the middle of a module and therefore we need to use a hard link.